debug/feature/Tool-Config/bespoke.html

```diff
@@ -0,0 +1,65 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <title>Bespoke</title>
+
+        <script src="../../../dist/smk.js"></script>
+        <link rel="stylesheet" type="text/css" href="style.css"/>
+    </head>
+    <body>
+        <script src="default-config.js"></script>
+<!--***************************************************************************
+    This example shows a simple Bespoke tool configuration. A template has 
+    custom HTML. This also includes a Javascript function that is called by
+    a button in the custom HTML.
+-->
+    <div id="smk-map-frame"></div>
+    <script>
+        SMK.INIT( {
+            containerSel: "#smk-map-frame",
+            config: defaultConfig.concat( {
+                tools: [
+                    {
+                        "order": 1,
+                        "position": "toolbar",
+                        "icon": "now_widgets",
+                        "title": "Bespoke Tool",
+                        "useComponent": true,
+                        "component": {
+                            "template": 
+                            `
+                            <!-- All of a template's content must be inside a 
+                            pair of <div> tags. -->
```

Comment:
Not quite true, they don't have to be div tags, there just needs to be a single set of enclosing parent tags (for example, you might be using a component here).